native/packages/react-native-bpk-component-button/src/layout/BpkButtonContainer.android.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  TouchableNativeFeedback,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  getStyleForElement,
+  getAndroidBackgroundColour,
+  themePropType,
+} from './../utils';
+
+const BpkButtonContainer = (props) => {
+  const theme = props.theme;
```

Comment:
The new eslitn rules will complain about this not being destructured. Why aren't we doing all the destructing in a single place anyway?

Reply:
After a chat with @matthewdavidson it looks better is the props object is not used after has been destructured, if the new eslint will complain about it, we'll have to trade in some readability, to make the linter happy 😄